README.md

```diff
@@ -13,6 +13,64 @@ The Intelligent Kiosk Sample is a collection of demos showcasing workflows and e
 2. Right click on the IntelligentKioskSample project and set it as the StartUp Project 
 3. Run the solution (F5)
 4. Enter your API Keys in the Settings page (they will be saved in the user profile). See [Settings](Documentation/AppSettings.md) for  more details on the available settings.
+  * If you prefer using command line you can execute the following azure cli commands from bash / git bash, to generate the cognitive services keys required for the Kiosk app
```

Comment:
Rather than putting the steps and example output in here, can you move it to a separate .md file, and reference it instead? Something like:  
* If you prefer using command line to obtain the keys, you can follow the steps in the in the <KeyAcquisitionScript.md reference> file on how to do that through azure cli commands from bash / git bash.

Reply:
Change incorporated